src/app/machines/views/MachineList/MachineListControls/MachinesFilterAccordion/MachinesFilterAccordion.tsx

```diff
@@ -12,6 +12,7 @@ import { FilterGroupKey } from "app/store/machine/types";
 export enum Label {
   Toggle = "Filters",
   Arch = "Architecture",
+  Deploy = "Deployment Target",
```

Comment:
Might be better to use a full name for clarity (and consistency with the type you created).

```suggestion
  DeploymentTarget = "Deployment Target",
```